### ISSUE TITLE
Adding a country results in weird UI Behavior

### ISSUE THREAD

**Issue by camgnostic** [mercycorps/TolaActivity#1367]
To reproduce:
Go to Admin->Countries
Click "Add new country"
fill out name, description, country code
click "Save"
Notice it appears briefly at the top of the country list
Then it disappears
Now it's gone from filters and list until you refresh the page.

**Comment by jennyhavah** [comment]
If you can always work around this by refreshing the page, then I would consider this a usability issue we ought to fix, but not a release blocker.

@hannalcamp, can you please check out this behavior and let us know if you consider it to be a release blocker? If not, I suggest we move it to Poblano.

**Comment by jennyhavah** [comment]
This may or may not be an example of the "lack of reactivity" described in #1366.

**Comment by RickieT** [comment]
I was able to reproduce this issue by following the steps to create then save, but when I clicked it at the top of the list, it disappeared as described. @jennyhavah its not ideal behavior, but it doesn't block the user from creating the country/editing/searching for it, so in my opinion yes we should fix, but I agree it might not be at release blocker level

**Comment by jennyhavah** [comment]
Oddly, I cannot reproduce the behavior of the country disappearing from the top of the list. I do see that it doesn't appear in the filters.

I'm going to defer to @hannalcamp to check out the behavior for herself and let us know if it's a release blocker.

**Comment by hannalcamp** [comment]
I agree, I wouldn't consider this a release blocker. Adding a country will for now be an extremely rare thing and be done only by Super Admins.

**Comment by jennyhavah** [comment]
Great, moving to Poblano.
